components/http_foundation/sessions.rst

```diff
@@ -190,8 +190,12 @@ data is an array, for example a set of tokens. In this case, managing the array
 becomes a burden because you have to retrieve the array then process it and
 store it again::
 
-    $tokens = array('tokens' => array('a' => 'a6c1e0b6',
-                                      'b' => 'f4a7b1f3'));
+    $tokens = array(
+        'tokens' => array(
+            'a' => 'a6c1e0b6',
+            'b' => 'f4a7b1f3',
+        )
```

Comment:
-1 for that proposal
+1 for the comma

I think the key is a bit lost in the proposal and it isn't very clear that it's a tree structure and a and b are childs of tokens.